contributors/guide/owners.md

```diff
@@ -115,6 +115,35 @@ emeritus_approvers:
 - emily    # 2019-01-05
 ```
 
+#### Cleanup
```

Comment:
what may help is to make reviewers list more dynamic, while applying very conservative metrics to approvers. It may also help if bot automation will suggest reviewers before suggesting approvers. Today PRs are saying to assign to the approver and suggest the approver. Also reviewers stats might be split by repos. 

wdyt?

Reply:
The reviewers that are assigned are from the reviewer pool, ideally the workflow pattern is: Reviewer assigned to PR reviews and applies lgtm and/or assigns the approver. If they did not assign and just applied lgtm then the PR author assigns the approvers listed.

Reply:
if there's a todo from this convo, please open an issue and we can deal with it there. thanks!